packages/web-components/src/components/content-switcher/content-switcher.ts

```diff
@@ -217,6 +245,18 @@ export default class CDSContentSwitcher extends LitElement {
   @property({ reflect: true })
   value = '';
 
+  /**
+   * Specify a selected index for the initially selected content
+   */
+  @property({ type: Number })
+  selectedIndex = 0;
+
+  /**
+   * Choose whether or not to automatically change selection on focus when left/right arrow pressed. Defaults to 'automatic'
+   */
+  @property()
```

Comment:
```suggestion
  @property({ attribute: 'selection-mode'})
```
and should we maybe a defs file with enums for the content switcher types?

Reply:
I'm not sure about this one. Do we have any examples of what you are mentioning?

Reply:
like this: https://github.com/carbon-design-system/carbon/blob/main/packages/web-components/src/components/accordion/defs.ts